tests/jobs/test_local_task_job.py

```diff
@@ -24,10 +24,11 @@
 import pytest
 from mock import patch
 
-from airflow import AirflowException, models, settings
+from airflow import settings
+from airflow.exceptions import AirflowException
 from airflow.executors.sequential_executor import SequentialExecutor
 from airflow.jobs import LocalTaskJob
-from airflow.models import DAG, TaskInstance as TI
+from airflow.models import DAG, DagBag, TaskInstance as TI
```

Comment:
Should we import DAG explicitly?

Reply:
Yes I think that would be good for consistency.

Would it be worth considering a similar change to `airflow/models/__init__.py`? Removing the implicit imports of models?

Reply:
This is a bit tricky, because we are relying on it in "resetdb" case. The models when imported register themselves in sqlalchemy and resetdb uses that information to know which tables to delete. However we already have some exceptions to that - some models are moved out of models package __init__.py and have to be added manually to resetdb. 

I think what would make more sense is to have a separate models/all_models.py which would import all models and this one can be used by resetdb,

But I think we should make a separate PR for that.

Reply:
I created an issue for that https://issues.apache.org/jira/browse/AIRFLOW-6870

Reply:
> I think what would make more sense is to have a separate models/all_models.py which would import all models and this one can be used by resetdb,

I am happy to do that as I am pylinting core code :D